src/controllers/job/Transactions.ts

```diff
@@ -1,12 +1,46 @@
 import { Request, Response } from 'express';
+import * as rp from 'request-promise';
 import * as Models from '../../models';
 
 const Stocks = Models.default.Stocks;
+const url = {
+  FPT:
+    'https://svr2.fireant.vn/api/Data/Markets/HistoricalQuotes?symbol=FPT&startDate=2020-1-20&endDate=2020-4-19'
```

Comment:
should be define a function:
 const urlCrawl = (options): string => {
   const { stockCode, startDate, endDate } = options;
  const endpoint = process.env.ENDPOINT_CRAWL_URL || 'https://svr2.fireant.vn/api/Data/Markets/HistoricalQuotes';

 return `${endpoint}/symbol=${stockCode}& startDate=${startDate}& endDate=${endDate};
}